### ISSUE TITLE
Fine-tune config

### ISSUE THREAD

**Issue by sunprinceS** [sunprinceS/MetaASR-CrossAccent#3]
During fine-tuning, in general we don't need warmup at first,
so which optimizer (maybe Adam, AdamW?) and which learning rate (same as inner-loop learn?) we should use?
